.rubocop.yml

```diff
@@ -41,3 +41,6 @@ RSpec/DescribeClass:
 
 RSpec/ExampleLength:
   Enabled: false
+
+Style/NumericPredicate:
```

Comment:
This is so that we can use `object.something > 0` like the rest of the world does.

Reply:
👍🏻